tensorflow/compiler/xla/service/hlo_verifier_test.cc

```diff
@@ -2348,6 +2347,57 @@ TEST_F(HloVerifierTest, ReduceScatterNonUniformGroups) {
               HasSubstr("Replica groups expected to be of uniform size"));
 }
 
+TEST_F(HloVerifierTest, ReduceScatterTwoTokens) {
```

Comment:
I think there is some confusion here between tokens and tuples. I think the intent is to add tuple support, so we should remove any mention of tokens in test names or in the change description.